src/avt/Database/CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ Formats/avtSTSDFileFormatInterface.C
 SET(GHOST_SOURCES
 Ghost/StructuredBoundary.C
 Ghost/avtDomainBoundaries.C
+Ghost/avtGhostNodeGenerator.C
```

Comment:
@brugger1 ...curious why this isn't done for the STMD, MTMD, MTSD? file formats also.

Reply:
@griffin28 The enhancement is independent of STSD, STMD, MTMD, MTSD  The class that implements the enhancement is in "avt/Database/Ghost". I'm not sure why the GitHub diff tool is showing it as part of "Formats/avtSTSDFileFormatInterface.C". A feature I guess.

Reply:
Ahhh ok...that's what I would expect.